src/utils/targeting/index.ts

```diff
@@ -0,0 +1,46 @@
+import { disableTargeting, enableTargeting } from '../../api/targeting'
+import inquirer from '../../ui/autocomplete'
+import { featurePrompt, EnvironmentPromptResult, environmentPrompt } from '../../ui/prompts'
+
+export const getResponsesForEnableDisableTargeting = async (
```

Comment:
Would this be more accurate as something like `getFeatureAndEnvironmentArgs`? I had to read the function code to understand what "responses" this is getting. I could also see us using this for targeting commands other than enable/disable